src/modalities/models/huggingface_adapters/hf_adapter.py

```diff
@@ -115,7 +116,7 @@ def forward(
             raise NotImplementedError
         model_input = {"input_ids": input_ids, "attention_mask": attention_mask}
         model_forward_output: dict[str, torch.Tensor] = self.model.forward(model_input)
-        if return_dict:
+        if not return_dict:
```

Comment:
I executed the checkpoint conversion. Everything works perfectly. 

I ran eval harness with these changes (since the HFModelAdapterConfig and HFModelAdapter are required while evaluating modalities using eval harness as well). This change seems to have affected the eval run. On reverting this change back, it seems to run.